Source/SVGIconVirtualImageList.pas

```diff
@@ -219,7 +219,6 @@ procedure TSVGIconVirtualImageList.SetCollection(const value: TSVGIconImageColle
 begin
   if FCollection <> Value then
   begin
-    FCollection := Value;
     if FCollection <> nil then
       FICollection.RemoveFreeNotification(Self);
```

Comment:
FICollection should be FCollection  (and below)

Reply:
Oops, not sure how that happened.